feature-staking-impl/src/main/java/jp/co/soramitsu/feature_staking_impl/presentation/staking/main/StakingViewModel.kt

```diff
@@ -79,6 +90,16 @@ class StakingViewModel(
         }
     }
 
+    val alertsFlow = getAlertsFlow().mapList { AlertModel.mapAlertToAlertModel(it, router) }.asLiveData()
+
+    private fun getAlertsFlow(): Flow<List<Alert>> {
+        val stakingState = runBlocking {
```

Comment:
How do you think we can get rid of runBlocking in here? To get current stash acc I have to call coroutine function 
@valentunn

Reply:
flatMapLatest on selectedAccountStakingStateFlow